File: frontend/src/aura-mobile/src/types/contractType.ts
export enum ContractType {
  Subscription = 0,
  Contract = 1,
  Membership = 2,
}

export const CONTRACT_TYPE_LABELS: Record<ContractType, string> = {
  [ContractType.Subscription]: 'Abo',
  [ContractType.Contract]: 'Vertrag',
  [ContractType.Membership]: 'Mitgliedschaft',
};

export const CONTRACT_TYPE_ICONS: Record<ContractType, string> = {
  [ContractType.Subscription]: '📺',
  [ContractType.Contract]: '📄',
  [ContractType.Membership]: '🎯',
};

File: backend/src/AuraContract.Core/Enums/ContractType.cs
namespace AuraContract.Core.Enums;

/// <summary>
/// Types of contracts/subscriptions
/// </summary>
public enum ContractType
{
    /// <summary>
    /// Subscription services (Netflix, Spotify, etc.)
    /// </summary>
    Subscription = 0,

    /// <summary>
    /// Long-term contracts (Rent, Insurance, Phone contracts, etc.)
    /// </summary>
    Contract = 1,

    /// <summary>
    /// Memberships (Gym, Clubs, etc.)
    /// </summary>
    Membership = 2
}
